### PR TITLE
Hopper Down.

### LINKED ISSUE #102: Hopper down
Button for hopper down (down dpad) and maybe a button for hopper up (up dpad) just for times when we want to run hopper up without shooter running

### PR DESCRIPTION
**Pull request by owenflatman**
close #102

### CODE REVIEW

**Comment by owenflatman** [comment]
Awaiting testing

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -130,6 +130,12 @@ private void configureButtonBindings() {
             .whenPressed(new FunctionalCommand(magazine::enable, () -> {
             }, interrupted -> magazine.disable(), () -> magazine.magSense.get(), magazine))
             .whenReleased(new InstantCommand(magazine::disable, magazine));
+        // Run hopper up with POV up (0)
+        new POVButton(operator, 0)
+            .whileHeld(new StartEndCommand(() -> magazine.enable(), () -> magazine.disable()));
+        // Run hopper up with POV down (180))
+        new POVButton(driver, 180).whileHeld(new StartEndCommand(
```
Comment (legoguy1000) [inline src/main/java/frc/robot/RobotContainer.java]: operator or driver??

Comment (owenflatman) [reply src/main/java/frc/robot/RobotContainer.java]: oops

File: src/main/java/frc/robot/subsystems/Magazine.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
  */
 
 public class Magazine extends PIDSubsystem {
-    private WPI_TalonFX magazineMotor = new WPI_TalonFX(Constants.Motors.magazineMotorID);
+    public WPI_TalonFX magazineMotor = new WPI_TalonFX(Constants.Motors.magazineMotorID);
```
Comment (legoguy1000) [inline src/main/java/frc/robot/subsystems/Magazine.java]: Don't make it public.  Just create a couple of methods in teh subsystem to turn on and off the motor (non-pid)

**Review by gvaldez7206 — APPROVED** [review]

**Pull request merged**